docs/modules/getting_started/pages/basic_walkthrough.adoc

```diff
@@ -38,13 +37,13 @@ If the issue persists, make sure the plugin is in the grafana plugins directory
 
 . In the *Config* tab, click *Enable*.
 +
-A *Disable* button indicates the plugin is already enabled.
-Skip to the next step (create <<custom actions>>).
+A *Disable* button indicates that the plugin is already enabled.
 You may also choose *Update* for the latest version of {page-component-title}.
```

Comment:
Does this update OpenNMS Horizon/Meridian itself, or the plugin that the user is setting up?

Reply:
Good point. Just the plugin.